module/VuFind/tests/integration-tests/src/VuFindTest/Mink/SearchLimitTest.php

```diff
@@ -67,7 +69,7 @@ protected function setUpLimitedSearch(
         $config = ['default_limit' => $default, 'limit_options' => $options];
         $this->changeConfigs(['searches' => ['General' => $config]]);
         $session = $this->getMinkSession();
-        $session->visit($this->getVuFindUrl() . '/Search/Results?limit=' . $limitParam);
+        $session->visit($this->getVuFindUrl() . '/Search/Results?lookfor=%22test publication%22&limit=' . $limitParam);
```

Comment:
Should we perhaps add the appropriate building filter here to limit to a single MARC file, just to ensure that future records are less likely to break the test?

Reply:
Makes sense. I replaced the search phrase with a building filter.